manifests/vhost.pp

```diff
@@ -1660,6 +1660,9 @@
 # @param ssl_proxyengine
 #   Specifies whether or not to use [SSLProxyEngine](https://httpd.apache.org/docs/current/mod/mod_ssl.html#sslproxyengine).
 #
+# @param ssl_require
+#   Sets the [SSLRequire](https://httpd.apache.org/docs/current/mod/mod_ssl.html#sslrequire) directive.
```

Comment:
This states:
> SSLRequire is deprecated and should in general be replaced by Require expr.

Is it really needed to add a deprecated option?

Reply:
If I can convince my users to use that, sure. Where/How do you set that, looking for the word "require" in a Puppet codebase brings up a few hits 😉

Reply:
I think they can be set via the `directories` parameter. It supports a lot of options and and one of them is `require`.

Reply:
I've removed that parameter now.